helm/prometheus-rules/templates/alerting-rules/deployment.workload-cluster.rules.yml

```diff
@@ -26,7 +26,7 @@ spec:
       annotations:
         description: '{{`Deployment {{ $labels.namespace}}/{{ $labels.deployment }} is not satisfied.`}}'
         opsrecipe: workload-cluster-deployment-not-satisfied/
-      expr: kube_deployment_status_replicas_unavailable{cluster_type="workload_cluster", deployment="metrics-server"} > 0
+      expr: kube_deployment_status_replicas_unavailable{cluster_type="workload_cluster", deployment=~"metrics-server|vertical-pod-autoscaler-app-admission-controller|vertical-pod-autoscaler-app-recommender|vertical-pod-autoscaler-app-updater"} > 0
```

Comment:
Why would Kaas teams be alerted when VPA does not run on a workload cluster ? Are we using it now as part of a release ?

Also, would it make sense to page when it's not working on Management clusters as well ?

Reply:
Okay I am confused:

Ludacris got this issue about owning VPA https://github.com/giantswarm/giantswarm/issues/17860

But according to the ownership doc it belongs to atlas:
https://intranet.giantswarm.io/docs/product/team-ownership/

What went wrong here? cc @puja108